teachopencadd/talktorials/T003_compound_unwanted_substructures/talktorial.ipynb

```diff
@@ -122,23 +122,17 @@
    "cell_type": "code",
```

Comment:
<p>I suggest to move the whole mols2grid topic to the end of the Substructure statistics section and look at the three most frequent substructures:</p><ul><li>Michael-acceptor <code>C=!@CC=[O,S]</code></li><li>Aliphatic-long-chain&nbsp;&nbsp;<code>[R0&amp;D2][R0&amp;D2][R0&amp;D2][R0&amp;D2]</code></li><li>Oxygen-nitrogen-single-bond <code>[OR0,NR0][OR0,NR0]</code></li></ul><p> </p><p>Some minor text edits:</p><p>Dataframes &gt; <code>DataFrame</code></p><p>unwanted substructures<strong>,</strong> let's have a look</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/279/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>